yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1261,7 +1262,6 @@ func init() {
 
 	analyzeSchemaCmd.Flags().StringVar(&targetDbVersionStrFlag, "target-db-version", "",
 		fmt.Sprintf("Target YugabyteDB version to analyze schema for. Defaults to latest stable version (%s)", ybversion.LatestStable.String()))
```

Comment:
> It only supports versions with 4 segments (A.B.C.D). No build number is allowed.


Should we mention this format of version to be provided in the flag?

Reply:
Makes sense, thanks! done.